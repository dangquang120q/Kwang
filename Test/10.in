59
11
1731620 5569496 2248209 8022091 6261645 8993549 7422165 6765729 2137922 9145761 7398314 
83
2367852 5753242 4730552 6400281 7374363 4535376 8137730 8328217 9460772 5916637 5435707 7637861 3226985 876302 8913223 7683664 9184298 9121652 4709124 2479552 230942 6036530 1710446 6529976 5435441 2516462 6462221 6154417 7098870 8465440 1206851 3286774 8871839 8863976 3477115 1810404 9287922 9510424 5014246 369050 9975059 3287692 8978759 2405038 7560383 1690135 1721039 5231264 4155361 4687068 4394714 9425352 6885835 457132 808432 7379662 5466295 254214 7592050 7809756 704870 3054126 8763181 4755304 8531893 404133 2249359 7142682 6559579 2485405 9867590 8872054 9054131 1254866 2900038 6808798 8163867 8059778 802482 8055766 4787808 1093648 5232021 
24
487091 8848811 1976856 3982581 6635788 4428541 5393178 4718824 8011093 1039252 9283714 333090 8371168 4198007 1429645 4702356 5235368 4232204 7171922 4353596 5123674 4776288 4574290 641734 
97
646905 1779393 7236786 7761021 1435564 6572526 5856945 8764236 173837 7325004 6274583 4408748 890867 3260144 8739816 1215010 9500160 9129469 5282834 8406464 6808938 5825712 4403851 3236607 3989487 2589503 546132 1290101 8798293 3665524 207282 8649734 7149139 8468157 8130551 6125916 3762326 9503836 2435727 8234350 4035964 1246237 7582003 2584527 8368032 7132638 2107124 6350774 2763446 257729 7351850 8208976 5764748 6758727 2963893 4918716 9987876 6995502 3151367 4639440 9647934 2311618 8039684 6329807 3836982 9939320 6385573 4598059 546186 6055299 3538053 561528 7221799 8088854 7652098 500411 4204912 343427 7442774 9474378 8554895 2136037 2363058 9215953 5599996 3878548 8275662 4277786 1815277 6163950 2797828 6203962 6101853 5089420 3335999 5145446 5454387 
78
577594 8263330 2992297 5824114 9217532 8722175 7680388 2789708 1473132 2268034 9960955 3359126 445842 3262845 3437944 4437409 6827982 2171256 2398003 3768231 105411 2040331 9120804 7811175 5117173 5357873 131752 6264948 2655077 3370239 8185899 7979197 5122502 7502798 1229074 6735069 719179 2656546 913897 5119763 7181363 1951159 7007368 8802378 8082140 4619603 2855381 3797785 8040778 9465343 9447301 9187835 490425 3219984 3483711 4437487 6035430 5465397 9575764 1357548 3110458 8748652 6913639 2058410 1034003 1035558 8335958 9644708 3041044 2111460 3143113 2487643 1289368 8443824 429503 5602783 3209809 6090127 
19
4206644 7960818 3136488 1625799 7716363 858700 8802928 5506794 934985 6703891 8300685 3889242 9853534 6478448 866885 4783693 7490250 7011662 2917707 
58
7373644 8787054 7120855 2614324 3382024 6381630 621692 4621988 6225598 8321015 705940 7397015 8999028 269616 9843153 2947240 8192468 6286891 5119823 7296297 5182665 5169727 3072617 5764797 7521208 8875056 3658299 1081001 5803464 5173557 1388848 7310722 7925801 7639861 426901 2257488 6974287 2199362 1957206 3971058 3502730 4164512 7084920 9013319 1138893 193871 626942 7526817 9358418 758794 9967131 4741629 3801242 3048929 3072494 7878286 6529453 7710036 
69
8932878 3601168 8094597 7238359 8705511 9806021 2072581 8213590 6508909 7488288 9279031 5865626 6022372 154220 5671175 5836812 1211694 3526601 4022665 7887062 2150794 5731503 2589441 6645089 4416484 7612090 5670988 2823714 6520785 6465025 1967255 544068 5092013 7091447 5747870 8584220 6534264 3265253 3486031 9379819 1787256 1786245 3704578 3400367 8617636 6058595 9248606 9916040 2747281 5068230 2293469 2985308 641465 8221814 8233476 5865972 2953625 9857573 6258176 6789663 479443 6469080 2661454 9584522 2310773 2647651 5508598 7929313 7866552 
64
6539956 7492968 328142 9880996 3849037 3587255 1815630 8020553 6956111 1213103 624613 6845215 3704484 7674544 6012539 9970942 7455596 4462956 470992 9735472 8261224 3952432 6471900 7554402 2657416 116882 581845 8005213 5760096 1580787 6627873 2094645 8387682 5668800 8941135 5812020 2631522 7354777 7080849 5086596 4000453 8987262 5341666 6615741 5144399 232044 6652059 2352633 1479609 3868419 1411096 4371702 7090944 8498916 1832847 4405782 8762374 3648647 8127852 3413466 4442510 7248518 7813157 9904146 
3
579341 4083289 1487289 
20
6635186 4636475 7709707 1808713 6485116 5215402 7115648 5538587 9633112 7862181 2830878 9409574 6153459 6647200 5182337 7440923 6604394 5548560 1545304 5011614 
55
247079 2363578 3873824 8231332 6685611 1675386 9558754 2548470 4981724 9640459 2736309 5888038 5413378 1634234 766503 3659426 4204392 9389638 4958075 3304747 2438964 2161598 8044332 5032058 5711701 4050568 9555307 3594750 636453 5643872 8956327 7481596 671996 2737197 2752637 6995527 4158342 8092770 9248538 6282590 7469103 5290444 617037 4741495 4108452 8582153 1042459 9254080 1274749 7451193 6025487 1180511 4036308 2368592 6009894 
35
3811079 3801844 3870927 8639069 4690464 8999380 8838539 9143549 6789772 1964637 948416 1178029 4903996 7177502 2579956 7413760 7797401 3248005 2283043 6615027 7359769 7076344 1881156 2995240 6582521 4079905 9165657 5231164 1254920 7422085 302411 1707307 8263254 666689 7394553 
57
1168163 2981934 8938801 6714638 6422379 1040911 3462098 6964992 2772097 3423864 7440100 5163522 7367601 4311495 3095406 7610521 2703160 5015985 2797304 8816501 7816881 1463822 1672914 1767732 7114107 8223104 2840282 2016618 4359915 2802230 7754916 4046184 432344 5430384 2775566 3373743 6221410 8423261 8860540 9303036 9051033 7198576 3165269 1188537 4265044 105047 6224555 4241102 3719074 9577056 5468683 5881228 4133626 6246251 6027865 4404103 3731891 
100
9194990 5274690 9276096 8358528 7545424 957586 1980826 992512 6863377 6401780 1906649 5177183 9411076 5075746 2429781 7461620 4313029 8519231 2987972 4435429 368503 6354542 5836438 2699759 4383854 831822 8275444 7001824 6187883 5335880 8843576 1017472 2108828 8249863 3841022 8471780 6083008 714839 3186939 3112637 1878766 8418221 6563631 9962887 6308832 1522436 3248755 1827488 7957129 2949304 9976737 9859896 1125455 6753600 5790751 1968068 6108963 9874886 7681485 1474015 8618841 8159816 7527883 3868811 1581586 7720799 8415387 8087478 9379664 7545380 1748991 6490623 1171574 6096570 8373907 9724650 6162040 1679974 8696429 2824299 2187258 5197034 3660544 4596758 502105 1931524 2804694 6566688 3305810 7914146 6112457 5343100 8499954 2969068 2538242 7607749 5736080 888627 7996797 8085150 
85
4677837 3635790 3956977 9560041 7355373 8785482 6402980 410502 3881697 290616 1615724 4705108 2723738 4229239 9983557 9095817 1748140 4349876 2905489 4372648 4899957 7293204 8936066 6751221 5764074 7549418 5341651 672836 3459434 1577202 7802360 2077454 4256927 6425977 6877008 2069990 197743 481391 4777525 4839103 2401892 4604687 1264673 915015 8306222 9121267 9925628 2775633 7185681 7650657 4279856 5102001 9174689 6031885 5694811 2890908 8560183 1988276 332951 1531548 3240828 527535 8547011 2582492 1994988 8321557 4609991 2230717 4909453 3361322 924991 7352369 1342871 7807468 7249675 6843649 6641672 1248577 6365821 6772089 3200241 2157887 9291492 8159362 8238170 
93
9687015 502715 8583864 154771 616459 5652932 1820641 1069418 6454063 8180282 2225928 4549862 7436124 5896672 4910463 2318694 1141759 3636643 6553188 7546198 1804372 1976595 5569151 2291666 8699635 1846278 2723176 5485216 3448440 2438047 4139816 7051762 1549703 137060 9424197 9977254 2213171 5159543 8892429 4446579 4269263 6099199 942468 555248 5970060 2071390 9176782 6525581 1853777 7890411 3136279 7061011 6832687 7352056 6291703 2790231 513934 3753456 2921277 2260992 121490 3933033 1653250 738099 3959445 9812984 9022284 3437621 1258717 9686405 1700039 2780688 6221107 6755344 941033 4605059 8799199 6984872 3657815 6441712 5647382 3470876 6723333 9492290 9175579 5352728 796995 1848920 1193944 330513 9371819 7293274 3564924 
76
1508555 7156561 9866861 5909156 1353448 2083765 4656488 2717213 1034514 2864923 8802484 285386 7848954 1088893 6503919 9070727 1773436 7666336 3035927 9903278 2216919 5442690 3505204 8354615 7067748 8019027 6665147 5700095 5906949 2974912 3020385 6411907 8492692 4227977 4081309 2339040 7932263 5406599 905191 3575746 3908032 4468543 9682339 1904188 1589727 7895609 6939062 108986 4616516 7173880 9707372 1632599 5832099 163128 2674908 6982696 8348740 7372396 5747507 998581 3747877 2532948 5741500 8739075 5352041 6791227 9729861 9399870 4719059 1243059 8649677 8287496 8528404 4319092 8256553 7009571 
58
1124355 1799173 5219339 2353514 5905820 6152037 2083974 8921495 4969742 2212883 9347446 7167356 8902599 8559494 4146098 3778007 5233548 3163107 2133564 1093770 2742563 5231809 2599290 9068624 2419602 5547404 8001609 9682556 9478318 1519292 446359 9263879 936797 2148804 1968942 211541 7725374 4896546 5704918 730335 439484 8774347 2927684 7014686 9199238 8955326 6769815 4202074 3287046 5976037 4339659 5979098 9196751 1098566 7124684 488997 4563070 709890 
46
9206941 2889444 916331 9596401 7479435 7415692 3178147 4873238 7648624 817853 6083876 8540951 5604640 6164488 5919225 7361394 8347945 4848140 2445108 3687773 7194857 4846677 5747816 9481777 5916418 6139576 7109657 3890055 6748755 8016904 4633283 4218059 5925097 434282 9809663 8956816 1167131 5124810 7842710 1383938 4065640 1683088 2100231 536173 793581 8115306 
11
2342210 1414001 8870616 3241688 6782795 8503468 3063180 7818246 5345810 3325201 3194871 
57
4376390 6854364 9000232 1414594 2270392 6614039 4668649 5522763 3907940 1578043 3243152 6521533 7235130 7351218 4521862 8158665 183300 9698230 8553419 5426611 1496836 6465718 596429 6226314 9830525 5966693 3029590 7671850 4963894 2102476 2505403 8911316 4143550 8846957 8882272 317964 7152130 6266312 7919601 3643312 4442610 5526935 2312622 4585392 5885306 8882748 5121138 4785854 5496948 4558134 8712876 1855066 6371258 7436019 5740357 2476110 895815 
26
3944414 6343740 7211012 7718975 8429121 4058644 7290862 4167424 7254810 7851351 5837200 5324252 7527305 3032409 4101968 3156782 491991 3902461 3451440 6283056 3680396 1912610 8491544 7346548 4545072 9969136 
11
6277351 4112520 6921363 877364 7030878 8901917 3570314 4313272 5424515 928830 1023198 
41
3653645 6191971 8613527 4622763 1583713 3420308 309600 1005331 2621338 5152322 997686 8491754 3521034 4594675 4033081 7394569 3484481 1397419 8300709 3443667 4900795 318539 8645924 7159530 9081876 951401 4015661 7503674 9066696 8413948 8546615 5021144 2399014 9931964 311303 3410505 5933386 3567652 282733 641521 5489730 
44
113724 1827360 3536968 8381457 5032324 7456515 5937873 1447998 1070781 2683876 5710758 3926867 8532097 5778831 9174639 2481365 8951777 350163 5997753 1786849 701483 3919472 4934976 4245201 8775086 6310054 8648284 615232 3008595 6418039 9724730 9178711 3610625 2575917 369196 4924629 931548 1736743 2653855 1128231 3945282 4715067 5175148 4203583 
48
3071790 9841984 9674364 262043 6922572 1393478 7822587 7064786 2804358 6893252 636521 7837182 3427759 8605332 2914314 4660425 4728490 5259130 1835012 948777 7905793 4776153 4111856 9770342 833569 9821212 4075259 6014816 6382949 7476500 5732810 3227361 6028834 8074498 9562721 7838158 5407576 5977221 7016941 5390619 9593816 3599370 1887521 5617234 8725271 6574432 9054116 9197648 
80
9589451 202467 8006336 3560422 6782280 8619318 3369041 2670339 4822283 2379241 9171481 7544200 7830728 2966083 4697721 690961 3653155 4501774 9921227 6185633 2578178 3811075 1091856 3955875 4038130 3891873 4704831 6829727 4027979 8347266 6265622 7822684 3229037 519103 1201585 1262738 1639907 8475893 1671617 8196811 478613 461300 2444670 2917978 6245296 5333501 5664826 5792173 8235001 6982330 9461578 9403426 1995535 7172931 6948300 9265610 8941291 8917180 7447243 7028852 8499600 2538524 6550791 7506743 4335378 1813763 8319790 8669708 8903437 1631834 1148223 4568588 4275382 9834065 5659095 8642603 1645455 7337702 3268739 4817875 
21
6457993 6900655 4369240 3293610 2784115 1113758 8637778 9723993 5487698 6972721 9735362 1178106 3467273 2376710 9944185 1203876 9305040 8722619 1438370 3651688 3738166 
23
5496623 531146 9831500 5024138 3747543 669293 7146181 6615820 7078361 3994287 179332 3176032 4875607 5010375 3915513 8401856 5883206 9346963 4005962 8254964 343677 8209117 3392598 
47
4513249 8211355 5153482 7434118 9710432 5894914 7533369 9851843 5813220 7849157 1713259 9317646 2665174 2082470 8337633 688490 1923376 1571143 8739288 1163412 247023 9986955 5893015 9022854 2812932 8318337 8205643 8664468 7171642 3814665 650153 4534252 2016815 993675 2712780 4740003 2752297 2580218 5083768 3463324 1418801 1769746 8040890 1724489 768544 8276988 9293710 
60
6145759 1530622 5761290 628515 1019591 1061672 6625064 4830174 8048671 6848408 2392420 1071021 3889567 2025113 9333279 8674335 8021217 3640750 823324 721406 4364338 2556461 4721788 3763075 4815267 9691855 653842 8790921 1696274 8842782 6480132 362963 1097419 2479637 5968613 9489831 2663168 9251137 5329952 2270428 8128737 4528480 5721192 3126292 3559228 3970066 3818412 7625142 4296560 335922 7026831 7595274 3246900 4920531 4940392 2608895 9573258 1151534 5664497 7928710 
92
7837244 8613843 5415706 4595810 4517353 7406864 9516124 9190562 8503141 8179883 3062179 1448084 5249036 6618528 6102660 9179306 4443004 9049672 9033670 4439676 3164755 3468694 5544098 1999960 5370414 9057676 9288005 516051 5649797 1720881 8904772 4938188 5830115 8085900 8240295 3743716 5636284 1732591 3410002 8504836 6185016 8143242 3232197 5146298 8633215 3910865 3989064 4964787 2232398 1331065 3850051 7095803 6926060 355680 4457233 2130244 6604268 9469302 2971120 8351495 5011172 1908404 2637133 5756191 7675417 8719236 5570982 1633144 2258946 6727855 8686973 3066743 6700652 7353380 7813970 1458817 4061020 3882870 9405873 4060649 7998228 8968984 7552700 6068441 9729464 5180868 3959129 9806708 5669349 6434210 7869099 9886208 
14
3181177 2532902 9861793 1018165 2469257 503277 1014939 6834716 1709365 7633136 2843626 2360946 1268721 6671171 
3
6804599 3625160 3404355 
12
590309 5210011 5608743 7543418 1806456 4645389 4642369 9796294 7584791 6088813 4489237 7608924 
5
7403101 2341874 4668575 6193477 4032581 
78
3414299 9505353 7870387 9175393 3045629 529219 2403012 7080310 1537682 3506385 2358949 9275385 9035583 7786047 7985452 8443729 8068164 6463851 1453659 4440611 9743441 1284888 4305203 4571104 6298694 4410451 9459584 4211639 7789910 4044308 2161166 9942111 3039535 9577506 4093022 5326114 7355141 3333239 9541027 7601739 6852744 5535631 6419538 1317877 1916692 7043965 8150782 7411175 1632982 6970673 6723738 928450 7706616 7169671 9694121 4286423 1729492 5651505 5868742 7088916 2835698 5885935 8186775 1699411 223980 9711232 2441878 1918380 9482305 2801369 5101401 6258382 6326386 9131568 3539281 2040777 6444770 193471 
87
7129117 6146138 4571127 3565719 9447821 9144524 4054381 1876054 8606091 989092 9527648 2789219 1208538 5104431 5346503 9164694 4918027 2782386 6883089 3226732 2736539 3935626 3764728 1670918 8283111 5164489 5700518 4727628 2958360 6232186 8396181 7717401 9219566 5492013 6741906 3319185 1855595 6574263 6984842 2491599 5558020 9807327 717957 6750159 829106 7760990 3802683 6388890 5395887 6938977 8875377 7284244 5458276 3806255 6661381 3438317 9603422 4720379 5589035 4110408 3725627 6136588 2500741 8262730 3324899 1224031 1755927 5601310 833200 9458379 8717523 5142913 3189989 627846 4444360 4775082 454307 8294244 5355913 4346090 9402866 4447920 5733605 965757 1947682 8002819 1278362 
39
8611063 7411477 9811378 8438424 8790914 1606827 7990969 5567481 4632143 3378677 5997598 7484681 2676382 4009513 7718534 5022303 4459700 3856388 3343032 1644880 9662972 4325869 2951331 7947101 757278 4739105 3562514 8027010 209253 7161444 1529132 5290577 1800811 5106414 2524749 4558518 579246 2036799 6978602 
65
5033686 7967269 8066751 5631524 1715027 5329466 2572892 4881527 3947375 5486913 1743236 8510923 8842020 470824 1816954 7989252 8593671 1641770 993203 5423310 8510168 1024504 8171357 2755116 9441340 9074586 3730501 2639198 2472024 7231270 3103405 9378872 1347436 1102751 8078646 8959515 1148063 9666285 970286 109085 4368459 8510791 3245340 9985526 9291215 9942566 2648851 9401048 6529704 3538853 1738484 2192558 2312141 2553002 5909567 930989 7514463 6476348 8227438 6172851 2158102 1057759 4326233 7582147 3677128 
76
6553836 6296837 8989056 6125813 3925796 9206853 1075643 8340907 4390645 1444499 2433445 7276412 8123188 9842573 3972073 8597150 6165533 1440321 6550736 4620795 3146776 6923705 566024 6121669 4153816 7583116 6085515 7249661 5617191 1293609 7371245 2948716 6802976 7274672 6594144 8488352 5655386 4048928 2923798 4781099 7606159 5887302 5432870 6005013 1209474 2130162 869819 5651602 2410102 1824673 1400127 9681068 6102920 5155034 3961967 1187704 7811455 3766507 485253 8227232 6131494 6594221 3249402 4131788 1268083 1447174 6987243 2532423 5852071 6814193 876787 4986741 3276440 751048 5324472 4182959 
31
171732 5233210 6881844 260885 6433454 3616167 9422012 2828036 9137956 243583 7622612 7753700 2416091 9329016 2503235 3253641 1281859 5287964 8610898 9290060 2741144 9364127 4657136 6111312 3978630 5755829 590942 8719908 8865352 5723473 2803480 
5
6883465 4074384 975817 2347613 2653660 
97
618051 3382201 3165644 6816601 2978473 6877512 529176 4161314 9051404 6545967 5469129 7854776 3011841 3321415 6565988 238411 1585530 1872939 7691309 8774664 5538195 663341 7066099 5687813 6532078 8600624 3763395 3611766 6072922 7777661 9457989 3569356 618588 9907298 6254375 9306385 8743846 4305918 769713 5181604 7215685 9374265 5546638 3835463 4073987 5827420 6887685 4725828 9475471 9531138 881523 2324667 3745471 8793214 4506157 536711 8352351 3490985 2932858 9996232 2111242 7484895 9860642 5045515 4613230 1585943 8339074 3013735 3836623 5565759 9365847 6399019 1319096 4170334 3685348 1410608 7436856 3454007 2796715 6013891 1233328 4097578 8180772 5051039 2377139 5305909 6846273 2354419 1183793 3378858 2553861 6266367 1725937 3884759 4434874 3625828 7203055 
22
1830881 4645538 7746552 7484502 5864489 5275835 7609912 4538717 8393350 2537468 7355924 4808806 1318941 8744837 3465858 3243767 5813553 1614193 5031428 4822955 2500742 9577634 
25
2131879 5263731 4815994 6827399 8476838 2428648 3998595 913906 7366142 1641948 1630684 4569915 1246706 5224598 2179428 1137953 4837446 4154275 7440531 7285317 3543610 1707279 7197205 9402291 8994955 
46
2549710 7289884 1096725 1095004 6410101 6557699 5778531 4267151 5914569 1705247 6917964 4663861 6419533 3020563 2470441 4582264 3321291 8240682 675823 2587167 2503748 2003600 3773466 6214258 9353446 6833317 2411780 7733316 4180776 1624512 1512154 349030 9222324 8846230 5559617 4716781 4608069 3035425 2357143 2853001 7501518 4085954 2047902 7719574 8918157 6644592 
38
2995721 408805 9380187 6033377 4680792 8804170 6609202 2405527 3973012 4672236 5806197 3253473 1413648 2541683 3822426 2790393 2330999 8851925 4749675 8489605 6899751 8088774 5117293 6594194 2778173 3348546 3857096 2226070 5602649 9846038 8276428 1402657 651045 1543928 2465662 1525144 7369443 8591432 
7
9824626 4420533 5423232 8522668 252136 4097274 6614773 
77
7261148 5287125 2063041 7249846 7502262 4526395 5471037 9605784 2907931 1116841 7760530 8219505 2495185 6009013 5407692 3223557 9933456 9954508 3241092 2731371 2626633 2624028 3970728 5568282 2876911 8155732 9872081 1867898 3517168 9382807 6920996 7777993 1740625 2653730 7404384 4088407 9298888 7009141 451322 8733141 1876932 881491 6521583 6174641 6684943 7442543 6541650 9699152 4765326 3708916 9388308 9422045 6883547 5534153 6270319 9490447 5421986 7296493 5009434 1100509 9359534 1304179 4781936 6468096 5357967 8480824 3505547 6886215 7322867 9570365 1623474 6579357 1420300 4864630 4219810 8607172 1828460 
97
2043501 7328180 2269032 1847860 7498327 8680146 9807030 403133 8214623 926763 4775775 4767013 2889524 2583093 1502053 5686002 3656240 4197431 6445045 4810283 2027230 9611841 838039 4867991 2093386 3110012 3555454 7910126 9347495 2012880 7334037 6918277 756768 8740836 7252274 2048925 439442 6744111 826984 5739302 6601613 3568925 7445700 2341735 2682196 3843460 5689933 6182693 9433121 4936781 9374539 7937310 4629304 5171777 6917025 7472470 7797704 2969005 4719247 1294687 8868897 9958575 6301720 4778416 5703866 9462308 9509602 8277666 7398952 9885142 3790693 6056280 9791932 1037415 4514544 775278 3849155 7373761 8363850 5007870 5195750 438046 9333677 2100031 3265203 9782257 258610 7562886 804644 5650112 6469795 4131264 3472197 5790353 1456805 7235981 5034066 
79
9426720 3179520 5914410 2379607 2224807 3401945 9142493 2590566 6731171 8006001 887081 5562159 4215678 4811735 6466971 9964419 6660029 8118454 5722828 5237536 8171342 1836949 2290518 6312323 6141711 7218195 734654 4390079 9958833 2419310 1337531 4018096 9672302 9586627 7938920 2194540 6639253 1439139 8080828 4597200 6984751 4459287 3987817 8205051 5597133 258553 5612108 5838837 1666854 5421511 7406126 4089965 8287414 6185171 5603171 6615135 6671446 8079911 280930 6473727 3964702 4208792 2057460 346251 6342593 9754451 9399143 6277086 8737388 7511998 9171300 317276 9210204 1349857 5384703 9606119 9681374 5955763 1702932 
84
9633127 4237028 5551163 3219775 5022899 1280067 7991527 8024938 4954145 3607323 9805632 1487555 8133013 1248874 8197871 1082394 6142501 1946979 8507482 7009189 8199090 8049964 6748620 3046959 9135179 6482399 6902105 3336417 5142711 2129898 8733882 8934114 6593596 7644295 9489250 3464331 4195773 2195819 2062354 5443950 9559637 811753 9483159 265326 7252640 9443323 9362217 3136528 6976665 8576461 423692 6288779 3252093 9294371 4387133 6211787 1531724 4572261 1497100 2457362 8451725 9770768 1112693 9952084 5188069 7067588 5773865 8276039 428425 1322790 1247994 1587979 8421297 5759125 3862861 8493607 9583752 2078351 1362249 2897956 3229531 6657450 2100353 8257592 
62
7760293 3236390 4028124 4157370 7017327 8771365 8501829 5516623 975575 732947 4259466 6186485 3604681 6939285 2615974 323959 197944 2554388 6969955 2519611 6448930 1486251 5406094 6798529 716340 7342227 7944066 726959 1124633 5520945 1020267 6792142 8666481 9205473 3040386 9266173 2203096 2963571 4186203 1985624 8448950 379110 4183294 7548655 1062547 7773730 5271434 800680 4337695 5979029 1063698 6516167 2717361 2415846 2588840 5387873 2659250 1685963 2150805 7489505 2076774 279787 
38
5101755 1997557 7373551 2437137 7761961 6835186 7574764 9490481 2232179 2016578 6601306 8087807 7768917 8366745 8728502 2296928 8964987 9849917 4085994 3139967 5837945 5559112 8808712 8967142 4773193 7556460 1982394 2869455 1279835 4374500 1241925 8863678 4182628 5893018 1958157 7095978 9955484 8520474 
76
553951 225156 3305853 3421474 8632412 1326579 4083453 9847599 4484036 5211244 3839453 999671 8266780 1361872 7450704 2211634 3224459 1954860 3089332 3694408 1591846 9971057 5026225 6074344 6486599 1126282 5344811 6775134 4583143 1959917 8414621 142689 7207696 2320597 4749423 8004961 7425007 7085064 4551533 9921095 7681456 615209 5218571 6687795 3909246 2240124 1440167 4033707 5951774 4357234 2203651 2841490 5177933 9216763 4301034 7472437 9225819 2865553 2155465 2552542 2249257 5361098 4284770 2403367 1792978 9419627 6430625 9466119 1941565 100754 9871816 7710339 8058206 874837 9951081 6350375 
68
9822159 6667315 5507501 4962772 9321563 1949589 8919411 1722217 8208767 5798470 8439524 3909352 2564587 4516091 1781921 8187898 2102873 9256280 4206076 1251420 4598762 8288093 2726721 2422217 5892623 8266707 1536546 8244798 6405297 5246345 1065567 6931687 8135170 3799179 4577772 8129307 9535337 2618940 8498864 4279009 3812408 1648870 3400239 1995567 9102514 7541166 7628996 3345118 3382755 6755658 7625806 8815475 9367499 2128154 6350941 8877224 148022 2073548 6626090 496744 8232951 8105382 4988304 1263520 1647263 6288568 3800177 5887037 
74
6266093 6434351 3239417 7279358 663965 9032173 5329074 4506265 8985189 1157882 1890695 2024288 6822593 7011390 1973885 5574266 5254229 809411 2508175 419446 8555050 1831525 4976855 3590454 4005525 4045907 8381204 4060711 9470984 6637471 2125382 2663153 2500387 493084 3570311 437104 5576713 5181257 3581879 7703571 3021748 613757 8288505 6552849 6013983 8090378 9812817 6500644 2400972 4403437 1736084 1852314 6097276 7218407 5352106 9920279 5191633 5854320 8428886 6441242 540735 7591146 883350 8950591 9420155 9645432 2160946 3297412 9765609 7663649 1626350 9102764 6988035 3080376 
