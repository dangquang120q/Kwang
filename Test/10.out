26161401
29353073
26503693
29575130
29181427
26957147
29168702
29579634
29756084
6149919
26904867
28754520
27308586
27931125
29814510
29469226
29477253
29500000
28508320
28887841
25192330
28528987
26889801
22854158
28080536
28078080
29433538
29344743
29403540
27580319
29549230
28754944
29422380
24329645
13834114
24990009
18265153
29396784
28938397
28265264
29594377
28730494
28115155
13611509
29764172
26715821
26874084
27493927
28078150
24962067
29760045
29650747
29677703
29528484
27243011
29295882
29843233
28724995
29498705
